Add NstDatabase.xml entries and update baked in database

rdanbrook opened the pull request.

rdanbrook commented:
Let me review this, it looks good but I want to ensure nothing breaks inadvertantly.

orbea commented:
Thanks, I'm just trying to force myself to learn so I might of done something stupid...

BlazeEntertainment committed:
Add four score support for multiple games

Added 4 player support for:

- Spacey McRacey
- NNNNNN
- Arkade Rush
- Justice Duel (Mega Cat Studios)
- BMX Simulator (Codemasters)
- Way of the Exploding Fist (Piko Interactive 2018)

BlazeEntertainment committed:
Added NST support for MM, MM:SQ and Bobl 1.2 (MMC1)

rdanbrook committed:
Fix up NstDatabase.xml entries and update baked in database

inactive123 commented:
This replaces the previous PR?

rdanbrook commented:
Yes, this replaces the previous one and also updates the baked in copy.
